app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -35,7 +35,8 @@ case class Project(
   extent: Option[Projected[Geometry]] = None,
   manualOrder: Boolean = true,
   isSingleBand: Boolean = false,
-  singleBandOptions: Option[SingleBandOptions.Params]
+  singleBandOptions: Option[SingleBandOptions.Params],
+  defaultAnnotationGroup: Option[UUID]
```

Comment:
Why is this optional? Because projects without annotations aren't getting one yet (or maybe ever)?

Reply:
pretty much. It only needs to be created when annotations are added to the project.